todo-mvc-kitchensink/src/outlets/TodoDetailsOutlet.ts

```diff
@@ -0,0 +1,13 @@
+import { Outlet } from '@dojo/routing/Outlet';
+import { Router } from '@dojo/routing/Router';
+
+import { TodoDetailsContainer } from './../containers/TodoDetailsContainer';
+
+export const TodoDetailsOutlet = Outlet(TodoDetailsContainer, 'edit', (params: any, type: string, location: string, router: Router<any>) => {
```

Comment:
Outlet for the route to edit a specific todo; injects a function to call to return to the parent outlet and also the id from the route params that is used by the TodoDetailsContainer